src/styles.js

```diff
@@ -0,0 +1,77 @@
+export const containerStyle = {
+  display: 'flex',
+  flexDirection: 'row',
+  width: '100%',
+  height: '100%',
+  minHeight: 200,
+  backgroundColor: '#ffffff',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+};
```

Comment:
instead of using `as CSSProperties` everywhere, could we typecast this to be CSSProperties?

e.g.

`export const containerStyle: CSSProperties = ...`

or

`export const containerStyle = { ... } as CSSProperties`